utils/migration_tools/model_registry_to_runs/README.md

```diff
@@ -0,0 +1,76 @@
+
+## Instructions
+
+To use the script, follow these steps:
+
+1. Execute `model_to_run.py`.
+2. Enter the project name you want to copy the model metadata from, in the `WORKSPACE_NAME/PROJECT_NAME` format. Leave this prompt as black to use the `NEPTUNE_PROJECT` environment variable.
+3. Enter the number of workers to use to copy the metadata. Leave blank to use all available CPUs.
+4. The script will generate run logs in the same folder as `model_to_run.py`. You can modify this location by editing the `logging.basicConfig()` function.
+
+
+## Note
+
+There are a few things to keep in mind when using this script:
+
+- All models and model versions will be copied. It is not possible to filter them currently†.
+- Most of the namespaces from the model/model_versions will be retained in the runs, except for the following:
+  - `sys` namespace:
+    - The `state` field cannot be copied.
+    - The `description`, `name`, and `tags` fields are copied to the `sys` namespace in the new run.
+    - All other fields are copied to a new `old_sys` namespace in the new run.
+- The _Model Stage_ is currently copied to `old_sys/stage` field. Unlike the `sys/stage` field, this field cannot be updated from the Web App. If you want to be able to update the _Model Stage_ from the Web App, the script can be modified to copy the stage as _Tags_ instead. 
+- File metadata is temporarily stored in a `tmp_tmp_%Y%m%d%H%M%S` folder in the current directory. This folder is deleted after the script finishes running.
+- The relative time x-axis in copied charts is based on the `sys/creation_time` of the original runs. Since this field is read-only, the relative time will be negative in the copied charts, as the logging time occurred before the creation time of the new run.
+- The hash of tracked artifacts may change between the original and new runs.
+- Each file copied as a `FileSet` will have its file name prefixed with the namespace where it was stored in the original run. For example, if the original run has a file named `hello_neptune.py` stored in the `source_code/files` namespace, the corresponding file in the new run will be named `source_code/files/hello_neptune.py`.
+
+† Support for these can be added based on feedback
+
+## Post-Migration
+- The source object of a run can be identified using `sys/custom_run_id`.
+- The Model ID of each model and corresponding model versions is stored as a group tag in the run to allow you to group models and model versions together, as shown in the screenshot above.
+- Runs made from models and model versions have the the `model` and `model_version` tags added respectively.
+- Once the migration and any sanity checks are complete, the copied Model/Model Versions can be deleted from the Model Registry to reclaim space
+- This script can also be used a template to update your logging script to start logging model metadata to Runs instead of the Model Registry.
```

Comment:
```suggestion
- This script can also be used a template to update your logging script to start logging model metadata to runs instead of the model registry.
```